stumpy/core.py

```diff
@@ -679,15 +706,30 @@ def rolling_nanstd(a, w):
     w : numpy.ndarray
         The rolling window size
 
+    welford : bool, default False
+        When False (default), the computation is parallelized and the std of
+        each subsequence is calculated on its own. When `welford==True`, the
+        welford method is used to reduce the computing time at the cost of slight
```

Comment:
at the cost of slightly reduced precision

Reply:
I will revise this.